client/balancer.go

```diff
@@ -0,0 +1,102 @@
+// Copyright 2018 The eth-indexer Authors
+// This file is part of the eth-indexer library.
+//
+// The eth-indexer library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The eth-indexer library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the eth-indexer library. If not, see <http://www.gnu.org/licenses/>.
+
+package client
+
+import (
+	"context"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum"
+	ethCommon "github.com/ethereum/go-ethereum/common"
+	"github.com/getamis/eth-indexer/contracts"
+	"github.com/getamis/eth-indexer/model"
+)
+
+//go:generate mockery -name Balancer
+// Balancer is a wrapper interface to batch get balances
+type Balancer interface {
+	// BalanceOf returns the ERC20/Ether balances
+	BalanceOf(context.Context, *big.Int, map[ethCommon.Address]map[ethCommon.Address]struct{}) (map[ethCommon.Address]map[ethCommon.Address]*big.Int, error)
+}
+
+// BalanceOf returns the erc20 balances
```

Comment:
// BalanceOf returns the erc20 or eth balances

Reply:
OK